cmd/nerdctl/build_test.go

```diff
@@ -202,6 +204,16 @@ COPY %s /`,
 	data, err := os.ReadFile(testFilePath)
 	assert.NilError(t, err)
 	assert.Equal(t, string(data), testContent)
+
+	aliasOutputDir := t.TempDir()
+	testAliasFilePath := filepath.Join(aliasOutputDir, testFileName)
+	base.Cmd("build", "-o", aliasOutputDir, buildCtx).AssertOK()
+	if _, err := os.Stat(testAliasFilePath); err != nil {
+		t.Fatal(err)
```

Comment:
Any reason not to use `assert.NilError` ?

Reply:
Well, I was planning to fix it.